### PR TITLE
Add sys adm env

### PR DESCRIPTION
**Pull request by riuvshin**
Add possibility to define system admin

### CODE REVIEW

**Comment by ibuziuk** [comment]
[test]

File: openshift/rh-che.config.yaml [hunk 1]
```diff
@@ -32,6 +32,7 @@ data:
   che-openshift-precreate-subpaths: "false"
   che-openshift-secure-routes: "true"
   che.predefined.stacks.reload_on_start: "true"
+  che-system-admin-name: "admin"
```
Comment (garagatyi) [inline openshift/rh-che.config.yaml]: Do we have a user with this name already?

Comment (ibuziuk) [reply openshift/rh-che.config.yaml]: I believe it would be applied via cm "on dsaas / dsaas-stg"

Comment (garagatyi) [reply openshift/rh-che.config.yaml]: yeah, you are right. It doesn't have to match this one

**Comment by ibuziuk** [comment]
[test]

File: openshift/rh-che.config.yaml (hunk 1 above)
Comment (rhopp) [reply openshift/rh-che.config.yaml]: Is it a problem, if such user does not exist? If so, it could be a problem for PR check deployment.

Comment (ibuziuk) [reply openshift/rh-che.config.yaml]: no, I believe there should not be any problems if the user does not exist - no admin user is going to be assigned.

Comment (riuvshin) [reply openshift/rh-che.config.yaml]: no, but I set it to be matching [this](https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties#L27)

**Pull request merged**